ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -78,6 +79,26 @@
   // the count of renegotiation after initial handshake done
   public final Counter sslRenegotiationCount;
 
```

Comment:
Line 29. Even until now I do not have a clear idea what a `network layer` refers to. Would be good to specify which part (or what is the focus) for this metrics set.

Line 48. Is `numActiveConnections` common to all selectors? If so, my feeling is that this metric would not be that useful.

Line 35. Is this metrics decremented only in Selector.java line 543? Any exception scenarios would also remove an inflight `send`?